src/main/java/karuna/karuna_backend/visitor/message/domain/VisitorMessageService.java

```diff
@@ -19,4 +24,16 @@ public VisitorMessageDto sendMessage(VisitorMessageCreateDto visitorMessageCreat
         emailSender.sendEmail("HARDCODED-EMAIL@EMAIL.PL", "KARUN VISTOR MESSAGE", savedVisitorMessage.getBody(), null);
         return VisitorMessageMapper.toDto(savedVisitorMessage);
     }
+
+
+    public VisitorMessageWrapper getMessages(VisitorMessageRequest visitorMessageRequest) {
+        PageRequest pageRequest = PageRequest.of(visitorMessageRequest.offset(), visitorMessageRequest.limit());
```

Comment:
We duplicate some fields from pagerequest, what if pagerequst will be fragmnt of pagerequest? @BarSwi @KrystynaCal

Reply:
@BarSwi what u think?

Reply:
What exactly do you mean by duplicate and "pagerequest will be fragment of pagerequest"

Reply:
I want to move pagerequest into visitormessagerequst, and remove offset and limit 🗡️  i write it chaotic, my bad

Reply:
I fix this. Hope is ok :)